README.md

```diff
@@ -1,4 +1,4 @@
-# Internado
+# Internado [![Travis](https://travis-ci.com/ubclaunchpad/Internado.svg?branch=master)](https://travis-ci.com/ubclaunchpad/Internado)
```

Comment:
Is there a reason you're removing this? This is the status badge for our Travis build and we do want to display it on our README somewhere